### ISSUE TITLE
[RA1]Rewrite NFVI requirements linked to SDN Controller

### ISSUE THREAD

**Issue by collivier** [cntt-n/CNTT#1075]
I would rather consider the networking solution. SDN Controller/solution mostly refers to a specific Neutron implementation where an SDN/Network controller is selected:
- req.inf.ntw.12
- req.inf.ntw.13
- req.inf.ntw.14

**Comment by collivier** [comment]
CC: @mf4716 who highlighted this requirements

**Comment by ASawwaf** [comment]
@collivier 
plz refer to https://github.com/cntt-n/CNTT/issues/1037#issuecomment-586672141 for more Networking part

**Comment by collivier** [comment]
@ASawwaf Thank you.

**Comment by pgoyal01** [comment]
The current requirements listed by @collivier are stated as follows:

| req.inf.ntw.12 | Network | The SDN solution should be configurable via orchestration or VIM systems in an automated manner using openly published API definitions. |  
|-- | -- | -- | 
| req.inf.ntw.13 | Network | The SDN solution should be able to support federated networks. |  
| req.inf.ntw.14 | Network | The SDN solution should be able to be centrally administrated and configured. |  

Nothing in the RM Networking Topic Issue #1037 and PR #1175 to-date invalidates these requirements or requires a changes to them; these requirements may need to be augmented with others as the RM Networking write-up evolves.

May I suggest that we assign this issue to Backlog, since at this time we do not have the RM Networking content and  we are unlikely to have time to address this issue (within the Baldy release) when the content does become available

**Comment by ASawwaf** [comment]
@pgoyal01 , I am fine with the above req. till we develop more in RM 

@walterkozlowski to comment

**Comment by pgoyal01** [comment]
@collivier Is it OK to assign this issue to Backlog?

**Comment by walterkozlowski** [comment]
@pgoyal01 @ASawwaf @collivier I have asked @sukhdevkapur to propose a plan to  address his concerns regarding short term SDN related actions needed.  I will wait for @sukhdevkapur response.

**Comment by hdamker** [comment]
Can someone explain to me what is the issue about? The initial description is not quite descriptive:

> I would rather consider the networking solution. SDN Controller/solution mostly refers to a specific Neutron implementation where an SDN/Network controller is selected:
> 
> * req.inf.ntw.12
> * req.inf.ntw.13
> * req.inf.ntw.14

Would be great if we can stick in RA1 close to the (OpenStack) community mainstream regarding networking.

**Comment by pgoyal01** [comment]
@hdamker It was suggested that we remove these 3 requirements pending the networking work in RM.

I proposed that, at the point in time when the RM Networking proposal has been finalised,  we can revisit these requirements (and possibly some others) and to move this issue to Backlog till that time.

**Comment by hdamker** [comment]
@pgoyal01 Fine for me to move to backlog. These requirements are currently only recommendations.

**Comment by walterkozlowski** [comment]
@pgoyal01 @hdamker I agree that we should push these requirements to backlog. Networking in RM is work in progress. As we agreed previously, once we have a stable Networking in RM (for the planning purposes, I declared yesterday it would be by the end of May 2020, TBC) we will start re-visiting RAx and RIx.  As I mentioned above, @sukhdevkapur proposes some important SDN-related topics that in his view can be taken care in parallel with the main NFG (Networking Focus Group) work in RM.  I requested @sukhdevkapur to prepare more specific plan, which we are going to discuss during next FNG meeting on Monday.

**Comment by collivier** [comment]
@hdamker @pgoyal01 @walterkozlowski @rabi-abdel @rabiabdel 
Yes I second removing this 3 requirements which I would consider as quite imprecise (possibly due to CNTT RM/RA1 model).
It works only in RA1 if we consider the neutron implementation to be compatible to the SDN paradigm.

I know there is already a requirement which forces the use of Neutron API  but "The SDN solution **should** be able to be centrally administrated and configured" may rather allow networking operations out of Neutron API.

Until we define any additional API operations which are not part of the classical OpenStack service operations, I won't see why there will be any SDN controller/solution specific requirements out of 
 - all SDN controllers to implement the Neutron API (it could be a simple rewrite of req.inf.ntw.14) 
 - Neutron API must be implementable by any SDN controller (ML2 requirements?

I would even consider us to fix baldy after any master change here.

**Comment by walterkozlowski** [comment]
@collivier, @hdamker, @pgoyal01, @sukhdevkapur, @scottsteinbrueck - Again, I back the move to remove these requirements form RA-1 until we have a more solid position in RM.  Since, this has been hanging around for some time, I propose @pgoyal01 to take an appropriate action in the RA-1 WS stream.

**Comment by karinesevilla** [comment]
I also agree to remove these 3 requirements (imprecise) and wait for RM developments
